lib/plugin/instance.rb

```diff
@@ -994,6 +994,21 @@ def register_notification_consolidation_plan(plan)
     DiscoursePluginRegistry.register_notification_consolidation_plan(plan, self)
   end
 
+  # Allows customizing existing topic-backed static pages, like:
+  # faq, tos, privacy (see: StaticController)
+  #
+  #   add_topic_static_page("faq") do |controller|
+  #     current_user&.locale == "pl" ? "polish_faq_topic_id" : "faq_topic_id"
+  #   end
```

Comment:
This looks good to me :+1: One suggestion though is that I think we need to explain here what the return value is supposed to be. In this case, the return value is the name of the `SiteSetting`.